plottable.d.ts

```diff
@@ -2132,6 +2132,19 @@ declare module Plottable {
              * @returns {Legend} The calling Legend
              */
             symbol(symbol: (datum: any, index: number) => SymbolFactory): Legend;
+            /**
+             * Gets the function determining the opacity of the symbols of the Legend.
+             *
+             * @returns {(datum: any, index: number) => number}
+             */
+            symbolOpacity(): (datum: any, index: number) => number;
+            /**
+             * Sets the function determining the opacity of the symbols of the Legend.
```

Comment:
Can we change the documentation to 
"Sets the opacity of the symbols of the Legend."